src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public abstract class Machine {
+    public abstract String doWork();
+    public abstract String stopWork();
```

Comment:
```suggestion
    public abstract String doWork();
    
    public abstract String stopWork();
```

methods should be separated from each other with an empty line